src/pages/workspaces/workspace/Analyses.js

```diff
@@ -224,8 +224,9 @@ const Analyses = _.flow(
   }),
   withViewToggle('analysesTab')
 )(({
-  apps, name: wsName, namespace, workspace, workspace: { accessLevel, canShare, workspace: { googleProject, bucketName } },
-  refreshApps, onRequesterPaysError, runtimes, persistentDisks, refreshRuntimes, appDataDisks
+  name: wsName, namespace, workspace, workspace: { accessLevel, canShare, workspace: { googleProject, bucketName } },
```

Comment:
I have put all the analyses-related items into an object to make it clearer who depends on those (as they are currently Google-specific).

These other classes use wrapWorkapce but do not depend on analysesData: Dashboard (workspace), Data, JobHistory, Workflows, SubmissionDetails, WorkflowDashboard, WorkflowView.